homework11/task1_metaclasses/tests/test_hw11_t1.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from project_hw11_task1.task1 import SimplifiedEnum
+
+
+def test_colors_enum():
+    class ColorsEnum(metaclass=SimplifiedEnum):
+        __keys = ("RED", "BLUE", "ORANGE", "BLACK")
+
+    a = ColorsEnum()
+    assert a.RED == "RED"
+    assert a.BLUE == "BLUE"
+    assert a.ORANGE == "ORANGE"
+    assert a.BLACK == "BLACK"
+    with pytest.raises(AttributeError):
+        a.GREEN
+
+
+def test_sizes_enum():
+    class SizesEnum(metaclass=SimplifiedEnum):
```

Comment:
`test_sizes_enum`  you put the name of the class into the test name.  Your code should work with user-defined classes so you should use the qualities of the class in the test name. 

For example:
`test_attributes_of_enum_are_accessible_via_instance_and_class`

So you can rename you tests:
`test_colors_enum` -> `test_attributes_of_enum_are_accessible_via_instance_and_class`
`test_sizes_enum` -> `test_attributes_of_enum_are_accessible_via_instance_and_class`

As you can notice names are the same. This is because these tests test the same functionality.